### ISSUE TITLE
All day events are returning the server date/time instead

### ISSUE THREAD

**Issue by djmtype** [soypunk/wp-plugin-google-calendar-events#1]
On all-day events, the server time/date are being returned instead. Either a string of `all day` should be returned or nothing at all.

The links on each of the events  are not linking out to the Google Event itself. Instead, they are linking internally, which obviously returns a 404.

**Comment by soypunk** [comment]
Thanks for the report @djmtype. The org I wrote this for did not use all day events so this wasn't a use case I had tested for. I'll see what I can do to whip up a check to handle this case as this would future proof the use of this plugin for the org in question... it is unlikely I'll add much more to the plugin as there's no budget left to do future improvements at this time.

**Comment by soypunk** [comment]
(and apologies in general for just how hacky this plugin is - it was designed to fit a pretty narrow set of requirements and under some money/time constraints so I had to take shortcuts to get things done. Contractor sad face.)

**Comment by soypunk** [comment]
I've written up a small patch for this - there are some code changes you may want to review that should make it easier to work with for your own needs in the future. all the best.

**Comment by djmtype** [comment]
Thanks @soypunk. I had a feeling this was more for a private client project. I was using something that seemed far too bloated and overkill for my needs and looking for an alternative.

**Comment by soypunk** [comment]
Probably the nasty hardcoded bit leftover is the route paths and the "add_rewrite_rule" stuff where it is set to a specific post # right now... fairly easily changed without having to get your hands terribly dirty. This was the first time I had worked WordPress plugins in about... five years? I mostly do Python/Django projects these days. If you have any other ideas for improvements, let me know! Like you I found most of the available plugins completely unsuitable to the org's needs - and many didn't support the new Google Calendar API which made them useless to begin with.
